packages/client/src/i18n/messages/views/verifyCertificate.ts

```diff
@@ -110,7 +110,7 @@ const messagesToDefine: IVerifyCertificateMessages = {
     defaultMessage: 'Name of registar',
     description: 'Label for name of registar center'
   },
-  createdAt: {
+  certifiedAt: {
     id: 'verifyCertificate.createdAt',
```

Comment:
```suggestion
    id: 'verifyCertificate.certifiedAt',
```
This should prevent confusion when only looking at the translation csv